.env.example

```
@@ -0,0 +1,4 @@
+PANEL_USERNAME = "admin"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)